exonum/src/helpers/mod.rs

```diff
@@ -156,6 +162,29 @@ pub fn byzantine_quorum(total: usize) -> usize {
     total * 2 / 3 + 1
 }
 
+/// Creates and initializes RustRuntime and GenesisConfig with information from collection of InstanceCollection.
+pub fn create_rust_runtime_and_genesis_config(
+    api_notifier: mpsc::Sender<UpdateEndpoints>,
+    consensus_config: ConsensusConfig,
+    instances: impl IntoIterator<Item = InstanceCollection>,
+) -> (RustRuntime, GenesisConfig) {
+    let (rust_runtime, genesis_config_builder) = instances.into_iter().fold(
```

Comment:
IMHO that construction is too complex for `fold`, I suggest to split it into several lines and use a loop.

Reply:
Yeah, for simple cases `fold` is better, I agree.